### PR TITLE
[DOCFIX] Fix namespace mgmt and storage mgmt docs

### PR DESCRIPTION
**Pull request by jiacheliu3**
1. Updated PropertyKey.java description for 1 key
2. Updated Namespace-Management.md fixing 1 indentation.

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -2001,7 +2001,8 @@ public String toString() {
               + "storage layer runs out of space. Valid options include "
               + "`alluxio.worker.block.evictor.LRFUEvictor`, "
               + "`alluxio.worker.block.evictor.GreedyEvictor`, "
-              + "`alluxio.worker.block.evictor.LRUEvictor`.")
+              + "`alluxio.worker.block.evictor.LRUEvictor`, "
+              + "`alluxio.worker.block.evictor.PartialLRUEvictor`.")
```
Comment (madanadit) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: why add this?

Comment (jiacheliu3) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: @madanadit I found `alluxio.worker.block.evictor.PartialLRUEvictor` is not included in the description in PropertyKey.java for this key so I added this to the description part.

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**